docs-developer/deploying.md

```diff
@@ -25,7 +25,7 @@ To be able to get the changed locales quickly, this command can be used
 (assuming that `upstream` is the remote you use for this repository):
 
 ```
-git fetch upstream && git diff --name-only upstream/main..upstream/l10n | awk -F '/' '{print $2}'
+git fetch upstream && git diff --name-only upstream/main..upstream/l10n | awk -F '/' '{printf $2 ", "}'; echo
```

Comment:
I think initially I was putting all the locales to a bullet list. That's why putting them to a new line made sense, so I can quickly select all and click on bullet list button on github to convert it a list. But I don't mind putting them in a single line.

(Unrelated: I honestly don't like this whole command even though I wrote it myself because it doesn't work perfectly all the time. In some cases it shows other directories too. I think it happens when people land something after l10n sync so it shows that PR's directory too. I will look at it and improve at some point.)

Reply:
I've seen this too, and I fixed it by using `...` instead of `..` and included this change in this PR initially, but I wasn't sure of myself, because according to the `gitrevisions` manpage we should have seen the opposite behavior... I didn't want to experiment more at that time, so I removed the change.